internal/cmd/errors.go

```diff
@@ -29,31 +35,27 @@ var (
 	ErrTLSNotVerified = errors.New(`The authenticity of the host can't be established.`)
 )
 
-// User facing variable errors
-type FailedLoginError struct {
-	LoggedInIdentity string
-	LoginMethod      loginMethod
```

Comment:
Neither of these fields add much value.

`LoggedInIdentity` could be useful to pass in context without needing a config lookup but `Error()` is doing a lookup anyway by comparing the passed in value with the config value. 

`LoginMethod` is used to determine the error message to output but it's set by the parent who already has this and more info about the error